templates/launch.html

```diff
@@ -48,6 +48,13 @@ <h1>Launch a survey</h1>
         </span>
     </div>
 
+    <div class="field-container">
```

Comment:
Should there not also be inputs for `case_id` and `account_service_url`?

Reply:
@nealedj We don't currently for tx_id which is essentially what case_id is so I don't think we need an input box here. The account_service_url is being used here to allow us to get back to launcher via the links given on log out etc so should be set within and not manually changed.

Reply:
I guess we should either:

1) Add _all_ fields that might be passed as part of the JWT payload to the form.
2) Exclude fields that are simply passed through and aren't used by runner

I don't mind which.

If 1) then it'd make sense to add `case_id`, `account_service_url` and `tx_id` (maybe more?)

If 2) then I think it'd make sense to remove `case_ref` and any other values that just get passed on. But we should probably still add `account_service_url` because that affects how runner behaves (changing the link URLs).

Reply:
If we removed the values would we still generate them in the background?

Reply:
@nealedj ^^^